### PR TITLE
Bugfix/GitHub issue 74 lapack test fail

### PR DESCRIPTION
**Pull request by balos1**

### CODE REVIEW

File: doc/arkode/guide/source/Introduction.rst [hunk 1]
```diff
@@ -126,6 +126,12 @@ Fixed a bug with the Kokkos interfaces that would arise when using clang.
 Fixed a compilation error with the Intel oneAPI 2022.2 Fortran compiler in the
 Fortran 2003 interface test for the serial ``N_Vector``.
 
+Fixed a bug in the SUNLINSOL_LAPACKBAND and SUNLINSOL_LAPACKDENSE modules
+which would cause the tests to fail on some platforms. 
+
+Fixed a bug in the SUNLINSOL_LAPACKBAND and SUNLINSOL_LAPACKDENSE modules
+which would cause the tests to fail on some platforms. 
+
```
Comment (gardner48) [inline doc/arkode/guide/source/Introduction.rst:134]:
```suggestion
```

**Review by gardner48 — CHANGES_REQUESTED** [review]
Minor doc fix, otherwise LGTM

**Review by gardner48 — APPROVED** [review]

File: examples/sunlinsol/lapackband/test_sunlinsol_lapackband.c [hunk 2]
```diff
@@ -135,13 +135,12 @@ int main(int argc, char *argv[])
   /* Print result */
   if (fails) {
     printf("FAIL: SUNLinSol module failed %i tests \n \n", fails);
-    printf("\nA (original) =\n");
-    SUNBandMatrix_Print(B,stdout);
```
Comment (drreynolds) [inline examples/sunlinsol/lapackband/test_sunlinsol_lapackband.c:139]: Was printing of the banded matrices causing the test failures, or were these removed for some other reason?

File: examples/sunlinsol/lapackdense/test_sunlinsol_lapackdense.c [hunk 3]
```diff
@@ -132,13 +132,12 @@ int main(int argc, char *argv[])
   /* Print result */
   if (fails) {
     printf("FAIL: SUNLinSol module failed %i tests \n \n", fails);
-    printf("\nA (original) =\n");
-    SUNDenseMatrix_Print(B,stdout);
```
Comment (drreynolds) [inline examples/sunlinsol/lapackdense/test_sunlinsol_lapackdense.c:136]: Was printing of the dense matrices causing the test failures, or were these removed for some other reason?

**Review by drreynolds — COMMENTED** [review]
Unless printing of the original/factored matrices when encountering a failure causes trouble, I would like for these to be added back in.

File: examples/sunlinsol/lapackdense/test_sunlinsol_lapackdense.c (hunk 3 above)
Comment (balos1) [reply examples/sunlinsol/lapackdense/test_sunlinsol_lapackdense.c:136]: It was not contributing to the failures, but it was making deciphering the log difficult.

File: examples/sunlinsol/lapackband/test_sunlinsol_lapackband.c (hunk 2 above)
Comment (balos1) [reply examples/sunlinsol/lapackband/test_sunlinsol_lapackband.c:139]: It was not contributing to the failures, but it was making deciphering the log difficult.

**Comment by balos1** [comment]
@drreynolds I am not sure what the advantage to having them printed is when this is a test of the linear solver not the matrices.

**Comment by drreynolds** [comment]
We would print both the original matrix and the factored matrix.  If the direct linear solver is somehow defective, then that factorization will have errors, and by looking at those we may be able to deduce the problem.  

The probability of an error is rather small, so I do not see any problem with having this extra output when a failure occurs.  Plus, if an error does arise, we would need to manually add these print statements back in to see what was wrong with the factorization.

**Comment by cswoodward** [comment]
It seems that Cody's suggestion of making this an option would be reasonable - it leaves the error output cleaner for gathering info on the initial failure, but then flipping it on gives info on the matrix factorization - which can be useful, as Dan notes.

**Pull request merged**